blt/urls.py

```diff
@@ -523,6 +524,7 @@
         ContributorStatsView.as_view(today=True),
         name="today-contributor-stats",
     ),
+    path("api/ask/", question_answer_view, name="ask"),
```

Comment:
Let's use something more meaningful:
```suggestion
    path("api/chatbot/conversation", chatbot_conversation, name="chatbot_conversation"),
```